docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -283,7 +283,158 @@ kube-dns-{network} shall be managed by Endpoints controller just like a regular
 
 __EndpointSlices__, the new type introduced in k8s v1.17, is out of current scope.
 
-### Ingress/egress/network policies
+### Network Policy
+
+Network policies, expressed as intents, are used to enforce desired network security within Arktos. Without any network policy in place, Arktos allows any pod to access other pods, across all tenants and namespaces for a flat typed network environment. VPC typed network environment, on the other hand, does provide strong tenant level network security even in the absence of any network policy in place.
+
+Existing k8s network policy specifications provide support for enforcing tenant scoped network security. Allowing access across tenant boundary is currently not supported within Arktos.
+
+Network-isolation mode makes more sense in most cases. Below is an example for specific network, my-network, in one namespace. Similar network policies need to apply to other namespaces, too.
```

Comment:
Suggest we remove "tenant: foo" from the YAML examples. They are not needed and we don't recommend putting it in YAML ( as it will limit which tenant can use this YAML file).

And also, can you help explicitly state in document:  is there any syntax change for network policy objects, or not changes at all?

Reply:
will remove "tenant: foo" as suggested.
I will make a explicit state that no change to existing k8s network policy spec.